src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -0,0 +1,467 @@
+  written with the assumption that you understand the concepts discussed in the
+  [Setup Signed Exchanges using a self-signed
+  certificate](/signed-exchanges-webpackager/#serve-sxgs-using-a-self-signed-certificate)
+  section.
+
+
+### Prerequisites
+
+  You have a `CanSignHttpExchanges` certificate. This
+  [page](https://github.com/google/webpackager/wiki/Certificate-Authorities) lists
+  the CAs that offer this type of certificate.
+
+
+### Generate a `CanSignHttpExchanges` certificate
+
+
+#### Prerequisites
+
+
+
+1. Install the
+   [`gen-certurl`](https://github.com/WICG/webpackage/blob/master/go/signedexchange/README.md)
+   tool.
+
+    ```shell
+    go get -v -u github.com/WICG/webpackage/go/signedexchange/cmd/gen-certurl
+    ```
+
+    `gen-certurl` is a tool that converts certificates to the certificate format
+    used by signed exchanges.
+
+
+
+2. Verify that `gen-certurl` has been installed correctly.
+
+    ```shell
+    gen-certurl --help
+    ```
+
+    This command should return information about its usage.
+
+
+#### Instructions
+
+
+
+1. Follow the steps 1 through 3 of [Generate a self-signed
+   certificate](/signed-exchanges-webpackager/#generate-a-self-signed-certificate).
+2. Convert `cert.pem` to the `application/cert-chain+cbor` format. 
+
+    ```shell
+    gen-certurl -pem cert.pem -ocsp &lt;(echo ocsp) > cert.cbor
+    ```
+
+    Certificates come in many formats. `cert.pem` is in the [PEM
+    format](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail). Certificates for
+    signed exchanges must be in the `application/cert-chain+cbor` format.
+
+    When using a self-signed certificate, `webpkgserver` automatically converts
+    the certificate indicated by the `PEMFile` option in `webpkgserver.toml` to
+    the `application/cert-chain+cbor` format. However, if you are using a
+    `CanSignHttpExchanges` certificate, you must generate the CBOR-encoded
+    certificate yourself.  
+
+
+### Setup up Web Packager for production use
+
+
+
+1. Follow steps 1 and 2 of [Setup the Web Packager Server for
+   testing](/signed-exchanges-webpackager/#setup-the-web-packager-server-for-testing).
+
+2. Open `webpkgserver.toml` with the editor of your choice and make the
+   following changes:
+    *   Change the line `PEMFile = cert.pem` to `PEMFile = cert.cbor`.
+
+
+### Upload the `CanSignHttpExchanges` certificate
+
+
+
+1.  Follow steps 1 through 4 of [Upload the self-signed
+    certificate](/signed-exchanges-webpackager/#upload-the-self-signed-certificate) to upload `cert.cbor`. 
+
+3. Adjust your server config to serve `cert.cbor` using the `Content-Type:
+   application/cert-chain+cbor` response header.
+
+    If this header is not set, you will see the following error when you inspect the
+    SXG in DevTools: `Content type of cert-url must be application/cert-chain+cbor.
+    Actual content type: text/html`.
+
+4. Restart `webpkgserver`.
+
+    ```shell
+    webpkgserver
+    ```
```

Comment:
As an optional step, you can also talk about how webpkgserver can auto-renew the SXG certs every 90 days if they have a DigiCert ACME account setup.  Relevant config items can be found here: https://github.com/google/webpackager/blob/master/cmd/webpkgserver/webpkgserver.example.toml#L228